packages/components-library/src/components/CartSelectionToast.vue

```diff
@@ -10,7 +10,7 @@
       <div v-if="previewToggle && value.length > 0" class="mb-4">
         <div class="row" v-for="(item, index) in value" :key="index">
           <div class="col">
-            {{item}}
+            {{ previewLabel(item) }}
```

Comment:
why not use a computed prop ?

Reply:
Computed properties don't (easily) have the ability to give it a parameter